rollup.config.js

```diff
@@ -2,21 +2,68 @@ import babel from 'rollup-plugin-babel'
 import css from 'rollup-plugin-css-porter'
 import pkg from './package.json'
 
+const fs = require('fs')
+
+/**
+ * Custom rollup plug-in which bundles all transforms' CSS in a single file.
+ * @param  {callback} options Rollup plug-in options func
+ * @return {void}
+ */
+const wikimediaPageLibraryTransformCSSRollupPlugin = (options = {}) => {
```

Comment:
It's cool that you were able to implement this in so few lines of code (and very instructive!)! Since this is a little unusual for the library and something we'll have to maintain, I'd like to keep a short decision record as I'm sure we'll have to explain it to someone at some point. Have you considered the following alternatives?

- Importing (or requiring) ThemeTransform.css in index.js before all other imports. I actually wouldn't mind moving all CSS imports to index.js since order matters.
- Moving ThemeTransform.css to live in wikimedia-page-library-override.css (right now it's just called Empty.css) and adding a note to the readme that this is expected to be linked prior to the transform CSS. These overrides are theme specific but they're also quite general and this would give us the opportunity to break these up across multiple files like Math.css, Infobox.css, FancyExtension.css, and so forth.
- If it's inserted alphabetically, renaming ThemeTransform.css something like _ThemeTransform.css.
- Generating a new CSS file for theming
- Switching to another plugin or bundler that supports this

Please consider moving this plugin to a separate file in our library and filing a PR or ticket against the old plugin's repo.